subprojects/docs/src/samples/build-organization/incubating/publishing-convention-plugins/kotlin/convention-plugins/src/main/kotlin/com.myorg.service-conventions.gradle.kts

```diff
@@ -0,0 +1,46 @@
+// Define conventions for service projects this organization.
+// Service projects need to use the organization's Java conventions and pass some additional checks
+
+// tag::plugins[]
+plugins {
+    id("com.myorg.java-conventions")
+}
+// end::plugins[]
+
+testing {
+    suites {
+        val test by getting(JvmTestSuite::class) {
+            useJUnitPlatform()
+
+            dependencies {
+                implementation("org.junit.jupiter:junit-jupiter:5.7.1")
+            }
+        }
+
+        val integrationTest by creating(JvmTestSuite::class) {
```

Comment:
we should be able to switch to registering when the other PR I mentioned above is merged